cosmosdb/client_endpoint.bal

```diff
@@ -317,14 +327,14 @@ public client class DataPlaneClient {
         return mapHeadersToResultType(response);
     }
 
-    # Execute a stored procedure.
+    # Executes a stored procedure.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container which contains the stored procedure
     # + storedProcedureId - ID of the stored procedure to execute
-    # + storedProcedureExecuteOptions - A record of type `StoredProcedureExecuteOptions` to specify the additional
-    #                            parameters
-    # + return - If successful, returns `json` with the output from the executed function. Else returns `Error`.
+    # + storedProcedureExecuteOptions - A record `cosmos_db:StoredProcedureExecuteOptions` to specify the 
```

Comment:
Suggestion: "A `cosmos_db:StoredProcedureExecuteOptions` record to specify the additional parameters"